users/models.py

```diff
@@ -212,6 +213,27 @@ def delete(self):
         for child in self:
             child.delete()
 
+    def has_accepted_communication_for_notification(self, notification_type: str):
+        """Filter guardians who have accepted communication of the type.
+        If the given notification type is included in the types that
+        needs an acceptance, the queryset will be filtered to guardians
+        who has True in `has_accepted_communication`.
+
+        Args:
+            notification_type (str, NotificationType constant string literals):
+                notification type.
+
+        Returns:
+            GuardianQuerySet: a filtered queryset if notification type needs acceptance,
+                otherwise returns self.
+
+        """
+        if not (
+            notification_type in notification_types_that_needs_communication_acceptance
+        ):
```

Comment:
Probably something will give a suggestion for this (Didn't run black on this so the formatting may change):
```suggestion
        if notification_type not in notification_types_that_needs_communication_acceptance:
```

I remember getting some suggestion by some tool about using `x not in y` instead of `not (x in y)`

Reply:
It's too long for 1 line (flake8 rule).

Reply:
So then it'll go to many lines, how does black make it? Is it like this:
```python
if (
    notification_type not in notification_types_that_need_communication_acceptance
):
```

Reply:
One way around the problem would be to make the variable name `notification_types_that_needs_communication_acceptance` shorter so then the `if notification_type not in <variable_name>:` could fit on one line, but this is a bit of painting the bike shed already, so as I said, do as you like